src/vs/workbench/browser/parts/editor/editorStatus.ts

```diff
@@ -556,7 +556,7 @@ export class EditorStatus extends Disposable implements IWorkbenchContribution {
 				// Compute number of lines selected.
 				let selectedNumberOfLines = (info.selections[0].endLineNumber - info.selections[0].startLineNumber + 1);
 
-				return format(nlsSingleSelectionRange, info.selections[0].positionLineNumber, info.selections[0].positionColumn, info.charactersSelected);
+				return format(nlsSingleSelectionRange, info.selections[0].positionLineNumber, info.selections[0].positionColumn, info.charactersSelected, selectedNumberOfLines);
```

Comment:
Maybe `selectedNumberOfLines` should be changed to `info.selectedNumberOfLines` for consistency